corehq/apps/app_manager/views/apps.py

```diff
@@ -490,11 +490,17 @@ def replace_all(list1, list2):
 
 @require_can_edit_apps
 @no_conflict_require_POST
-def edit_app_translations(request, domain, app_id):
+def edit_app_ui_translations(request, domain, app_id):
     params = json_request(request.POST)
     lang = params.get('lang')
     translations = params.get('translations')
     app = get_app(domain, app_id)
+
+    # Workaround for https://github.com/dimagi/commcare-hq/pull/10951#issuecomment-203978552
+    # auto-fill UI translations might have modules.m0 in the update originating from popular-translations docs
+    # since module.m0 is not a UI string, don't update modules.m0 in UI translations
+    translations.pop('modules.m0', None)
```

Comment:
:+1: I took a look through the view results and also found the following which should also be excluded:

```
forms.m0f0
 forms.m0f1
 forms.m0f2
 forms.m0f3
 m1.case_long.graph.x_title
 m1.case_long.graph1.y_axis_title
 m1.case_long.graph1.y_title
 m1.case_long.graph2.annotation1
 m1.case_long.graph2.annotation2
 m1.case_long.graph2.y_axis_title
 m1.case_long.graph2.y_title
 m1.case_long.graph3.x_axis_title
 m1.case_long.graph3.y_axis_title
 m1.case_long.graph3.y_title
 m1.case_long.graph4.y_title
 m1.case_long.graph5.y_title
 m1.case_long.tab1
 m1.case_long.tab2
 m1.case_long.tab3
 m1.case_short.callout.header
```

Might need to use a regex for some of them